modules/issue_tracker/jsx/editIssue.js

```diff
@@ -0,0 +1,628 @@
+/**
+ * Issue add/edit form
+ *
+ * Displays a form allowing a user to edit fields.
+ * Includes functionality for both adding a new issue
+ * and editing an existing issue.
+ *
+ * @author Caitrin Armstrong
+ * */
+
+
+var CollapsibleComment = React.createClass(
+    {
+
+        getInitialState: function () {
+            return {'collapsed': true}
+        },
+        toggleCollapsed: function () {
+            this.setState({'collapsed': !this.state.collapsed});
+        },
+        render: function () {
+
+            return (
+                <div className="row form-group">
+                    <div className="col-sm-9">
+                        <div className="btn btn-primary"
+                             onClick={this.toggleCollapsed}
+                             data-toggle="collapse"
+                             data-target="#comment-history"
+                        >
+                            Show Comment History
+                        </div>
+                    </div>
+                    <br></br>
+                    <div id="comment-history">
+                        <div className="col-sm-12">
+                            <div dangerouslySetInnerHTML={{__html: this.props.text}}/>
```

Comment:
@driusan Display HTML that is present inside `this.props.text`. 

It is not considered good practice to use it (#exploits), so I already showed a cleaner alternative to Caitrin.